### PR TITLE
French translation :fr:

### PR DESCRIPTION
**Pull request by thejsa**
Here is an initial translation into French :) :fr:

I am a first-year A-Level French student right now in the UK so there may be some mistakes or suboptimal wording, but hopefully this is of help ✨

### CODE REVIEW

File: content/definition/definition.fr.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
++++
+title = "La définition d'Ethical Source"
+aliases = ["/translations/fr"]
++++
+
+## La définition d'Ethical Source
+
+Un logiciel est considéré comme *Ethical Source* (source éthique) s'il satisfaire les cinq critère suivantes:
```
Comment (tchoutri) [inline content/definition/definition.fr.md]: Conjugaison manquante
```suggestion
Un logiciel est considéré comme *Ethical Source* (source éthique) s'il satisfait les cinq critère suivantes:
```

File: content/definition/definition.fr.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+1. Il est distribué librement avec code source et peut être utilisé ou combiné avec d’autres logiciels sans royalties ou frais.
```
Comment (tchoutri) [inline content/definition/definition.fr.md]: Ajout d'un adjectif possessif et de la recommandation officielle pour remplacer _royalties_ :
```suggestion
1. Il est distribué librement avec son code source et peut être utilisé ou combiné avec d’autres logiciels sans redevances ou frais.
```

File: content/definition/definition.fr.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+1. Il est développé en public et accueillir les apports de la communauté.
```
Comment (tchoutri) [inline content/definition/definition.fr.md]: Conjugaison manquante : 

```suggestion
1. Il est développé en public et accueille les apports de la communauté.
```

File: content/definition/definition.fr.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
++++
+
+## La définition d'Ethical Source
+
+Un logiciel est considéré comme *Ethical Source* (source éthique) s'il satisfaire les cinq critère suivantes:
+
+1. Il est distribué librement avec code source et peut être utilisé ou combiné avec d’autres logiciels sans royalties ou frais.
+1. Il est développé en public et accueillir les apports de la communauté. 
+1. Sa communauté est régie par un code de conduite qui est appliqué de manière constant et équitable.
+1. Ses créateur·rice·s ont le droit d’interdire son utilisation par les personnes ou les organisations qui ont impliqué aux violations des droits humains ou autres comportements qu’elles et ils juger être immoral.
```
Comment (tchoutri) [inline content/definition/definition.fr.md]: Rephrasage et conjugaison : 
```suggestion
1. Ses créateur·rice·s ont le droit d’interdire son utilisation par les personnes ou les organisations qui sont impliqué dans des violations des droits humains ou autres comportements qu’elles et ils jugent être immoraux.
```

File: content/definition/definition.fr.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+1. Ses créateur·rice·s ont le droit à solliciter de rémunération raisonnable et volontaire aux communautés ou institutions qui tire avantage du logiciel.
```
Comment (tchoutri) [inline content/definition/definition.fr.md]:
```suggestion
1. Ses créateur·rice·s ont le droit de solliciter une rémunération raisonnable et volontaire auprès des communautés ou institutions qui tirent avantage du logiciel.
```

File: content/definition/definition.fr.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+La définition d’Ethical Source est un document vivant. La communauté d’Ethical Source en général est propriétaire à cette définition et peut l’améliorer et évoluer en contribuant les tickets et révisions dans notre dépôt GitHub.
```
Comment (tchoutri) [inline content/definition/definition.fr.md]:
```suggestion
La définition d’*Ethical Source* est un document vivant. La communauté d’*Ethical Source* dans sa globalité est propriétaire de cette définition et peut l’améliorer et la faire évoluer en y contribuant, notamment, par des tickets et révisions dans notre dépôt GitHub.
```

**Review by tchoutri — CHANGES_REQUESTED** [review]
A few comments each time, otherwise very good job. The essence of the text was well-transmitted.

**Comment by thejsa** [comment]
@tchoutri :
> 
> 
> A few comments each time, otherwise very good job. The essence of the text was well-transmitted.

Ah looks like I missed a fair few conjugations; thanks for the adjustments and feedback, they all look great and help a lot. :)

**Comment by CoralineAda** [comment]
Thank you both so much!

**Pull request merged**